coordinator/config.go

```diff
@@ -32,6 +32,7 @@ type Config struct {
 	WriteTimeout         toml.Duration `toml:"write-timeout"`
 	MaxConcurrentQueries int           `toml:"max-concurrent-queries"`
 	QueryTimeout         toml.Duration `toml:"query-timeout"`
+	LogQueryTimeouts		 bool					 `toml:"log-query-timeouts"`
```

Comment:
Your code has not been ran through `gofmt`. We can't accept it until it passes `gofmt` as well as `go vet`. Our CI will ensure that `go vet` passes.